src/scalarstats.jl

```diff
@@ -66,6 +88,12 @@ function mode{T<:Integer}(a::AbstractArray{T}, rgn::UnitRange{T})
     return mv
 end
 
+"""
+    modes(a[, rng]) -> Vector
+
+Return all modes (most common numbers) of an array, optionally over a
```

Comment:
So this only returns several values when counts are tied? Probably worth mentioning.